docs/api-reference/index.rst

```diff
@@ -4,6 +4,56 @@ API Reference
 Warehouse has several API endpoints. See :doc:`../application` for the
 parts of Warehouse that generate them.
 
+.. contents:: :local:
+
+API Policies
+------------
+
+Please be aware of the following PyPI API Policies:
+
+Caching
+~~~~~~~
+
+All API requests are cached. Requests to the JSON, RSS or Legacy APIs are
+cached by our CDN provider. You can determine if you've hit the cache based on
+the ``X-Cache`` and ``X-Cache-Hits`` headers in the response.
+
+Requests to the JSON, RSS and Legacy APIs also provide an ``ETag`` header. If
+you're making a lot of repeated requests, please ensure your API consumer will
+respect this header to determine whether to actually repeat a request or not.
+
+The XML-RPC API does not have the ability to indicate cached responses.
+
+Rate Limiting
+~~~~~~~~~~~~~
+
+Due to the heavy caching and CDN use, there is currently no rate limiting of
+PyPI APIs at the edge. The XML-RPC API may be rate limited if usage is causing
+degradation of service.
+
+In addition, PyPI reserves the right to temporarily or permanently prohibit a
+consumer based on irresponsible activity.
+
+If you plan to make a lot of requests to a PyPI API, please adhere to
+the following suggestions:
+
+* Set your consumer's ``User-Agent`` header to uniquely identify your requests
+* Try not to make a lot of requests (thousands) in a short amount of time
+  (minutes). Generally PyPI can handle it, but it's preferred to make requests
+  in serial over a longer amount of time if possible.
```

Comment:
I suggest we add "consider running your own organizational mirror instead/in addition" & point to bandersnatch.

Reply:
I'm not sure that's good advice for folks that need consume our APIs. Does bandersnatch provide all the exact same APIs as PyPI?

Generally I think "use bandersnatch" is good advice for users that either can't use `pypi.org` or are concerned about it's availability. I'm not sure that using it could be considered a "best practice" for consuming our API.

(With the exception of using a mirror for the Simple/download API, which could be helpful in reducing load on our CDN, but I think we don't really have the assumption that this is necessary or recommended for big consumers -- we just handle the traffic.)

Reply:
This document tells API users how to be good neighbors, so I suggest we add a line advising orgs that it would be friendly for them to use internal mirrors for Simple/downloads. But my opinion is mild here.

Reply:
I found a happy medium 🙂